apps/entry/models.py

```diff
@@ -349,6 +349,20 @@ class Entry(MetaInformationArchiveAbstractModel, models.Model):
 
     # Properties
 
+    @property
+    def is_reviewed(self):
+        if not hasattr(self, '_is_reviewed'):
+            self._is_reviewed = self.reviewing.filter(status=EntryReviewer.REVIEW_STATUS.REVIEW_COMPLETED).exists()
+        return self._is_reviewed
```

Comment:
We should probably not cache the value for cases where we are not pre-calculating them as they may change (mostly when we are fetching a single instance and changing entry review). What do you think?
```suggestion
       # Use _is_reviewed from annotate if available.
        if not hasattr(self, '_is_reviewed'):
            return self.reviewing.filter(status=EntryReviewer.REVIEW_STATUS.REVIEW_COMPLETED).exists()
        return self._is_reviewed
```